test/filter/test_emoji.rb

```diff
@@ -1,7 +1,6 @@
 # ~*~ encoding: utf-8 ~*~
 path = File.join(File.dirname(__FILE__), "..", "helper")
 require File.expand_path(path)
-require 'ostruct'
```

Comment:
I first started what evolved into the `MockWiki` `mock_page` helpers in this file in a previous PR, but they ended up in `helpers.rb`. Just forgot to remove the `require` here. :)